### ISSUE TITLE
Link to release post will only work for snapshot reports

### ISSUE THREAD

**Issue by chandler05** [mojira/r-isa#31]
## The Bug
Just like #26 says, the value of the post link is hardcoded as https://www.minecraft.net/en-us/article/minecraft-snapshot-${currentSnapshot.name}
Which means full releases, pre releases, and release candidates will not work as they have different beginnings to their post links

Created as a separate issue because fixing this one does not necessarily mean fixing #26

**Comment by SPGoding** [comment]
I don't think it is possible to automate the process of getting post link; we might have to correct the link manually whenever it's wrong :thinking:
